content/riak/kv/2.0.0/developing/usage/conflict-resolution.md

```diff
@@ -541,11 +541,13 @@ obj.store
 bucket = client.bucket_type('siblings_allowed').bucket('nickolodeon')
 obj = bucket.get('best_character')
 
-# Then we modify the object's value
-new_obj.data = 'Stimpy'
+# Then pick one of the siblings
+resolved_sibling = obj.siblings[3]
 
-# Then we store the object, which has the vector clock already attached
-new_obj.store(vclock=vclock)
+# Then replace the siblings data structure with the single sibling we want to keep
+obj.siblings = [resolved_sibling]
+obj.store()
+obj.reload()
```

Comment:
Seems to work fine without `reload()` ([commit](https://github.com/basho/riak-python-client/commit/45c6091519e6bac1c034b53ffb41ce646d983007)). I'd say remove it from the examples.